tests/unit/amo/api/test_users.js

```diff
@@ -134,4 +135,33 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('deleteUserPicture', () => {
+    const mockResponse = () => createApiResponse({
```

Comment:
Why is this a function rather than just a variable? Also, I know you've only added the one test, but this `mockResponse` function is used a number of times in this file, so maybe it would be worthwhile to either move it to the top of the file, or have a `createFakeUserAccountResponse` in one of the helper files?

Reply:
mmh right, we could probably move this function to the top of the file.